app/code/community/Bolt/Boltpay/DuplicateTransitionException.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Bolt magento plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2018 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+/**
+ * Class Bolt_Boltpay_DuplicateTransitionException
+ *
+ * This exception is thrown when a transaction is trying to process order creation while a previous transaction already started before its arrival 
+ */
+class Bolt_Boltpay_DuplicateTransitionException extends Exception
```

Comment:
maybe DuplicateOrderException? 

We could have another duplicate (eg., receiving void hook twice for some reason), which we don't treat here